engine/src/main/java/org/terasology/persistence/serializers/PrefabSerializer.java

```diff
@@ -126,6 +114,7 @@ public PrefabData deserialize(EntityData.Prefab prefabData) {
     public PrefabData deserialize(EntityData.Prefab prefabData, List<EntityData.Prefab> deltas) {
         Module context = ModuleContext.getContext();
         PrefabData result = new PrefabData();
+        List<String> unknownComponents = new ArrayList<>();
```

Comment:
issue: this PR adds this but I don't see it used?

Reply:
nice catch... leftover of a previous attempt